mysql-test/suite/galera/t/galera_restart_replica.cnf

```diff
@@ -0,0 +1,31 @@
+!include ../galera_2x2nodes.cnf
+
```

Comment:
why using two clusters with 2 nodes? It appears that node_2 is not used in the test, so a topology of galera_2nodes_as_slave.cnf should be enough for this test, imo.

Reply:
good point